### PR TITLE
Docs and comments cleanup

### PR DESCRIPTION
**Pull request by jimklimov**
Make the playground tidier, so other PRs can concentrate just on their improvements :)

### CODE REVIEW

File: doc/znapzend.pod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-=head1 NAME
+[01;31m[K=head1 NAME[m[K
```
Comment (oetiker) [inline doc/znapzend.pod]:
```suggestion
=head1 NAME
```

File: doc/znapzendzetup.pod [hunk 2]
```diff
@@ -1,4 +1,4 @@
-=head1 NAME
+[01;31m[K=head1 NAME[m[K
```
Comment (oetiker) [inline doc/znapzendzetup.pod]:
```suggestion
=head1 NAME
```

File: doc/znapzendztatz.pod [hunk 3]
```diff
@@ -1,4 +1,4 @@
-=head1 NAME
+[01;31m[K=head1 NAME[m[K
```
Comment (oetiker) [inline doc/znapzendztatz.pod]: =head1 NAME

File: doc/znapzend.pod (hunk 1 above)
Comment (jimklimov) [reply doc/znapzend.pod]: Hmm, different tooling is differently eager it seems :)

**Pull request merged**